src/components/RecentListView.tsx

```diff
@@ -0,0 +1,93 @@
+import { DataProps } from "./inputArea";
+import { useState } from "react";
+import axios from "axios";
+import { baseUrl } from "../utils/baseUrl";
+
+export interface DataViewProps {
+    submit: DataProps;
+}
+
+interface CommentProps {
+    id?: number;
+    pasteBinId?: number;
+    comment: string;
+}
+
+export function DispleyRecentListView(props: DataViewProps): JSX.Element {
+    const [popupButton, setpopupButton] = useState<boolean>(false);
+    const [popup, setPopup] = useState<string>("");
+    const [viewButton, setViewButton] = useState<boolean>(false);
+    const [showAllComments, setShowAllComments] = useState<CommentProps[]>([]);
+
+    function handlePopupClick() {
+        setpopupButton(!popupButton);
```

Comment:
Consider having state popupButton to 'displayFullSummary' so that state can be true or false. Current naming is entertaining but confusing. 
Consider having (prev) => !prev instead